src/sentry/static/sentry/app/bootstrap.tsx

```diff
@@ -160,6 +168,8 @@ const globals = {
   // This is the primary entrypoint for rendering the sentry app.
   SentryRenderApp: () => render(Main),
 
+  RenderPipelineView,
```

Comment:
Could you toss a comment above this just indicating that we're using it for 'integration popups'